src/Uno.UI/Helpers/Xaml/UnoRuntimePlatformExtensions.cs

```diff
@@ -0,0 +1,23 @@
+namespace Uno.UI.Helpers;
+
+public static class UnoRuntimePlatformExtensions
```

Comment:
Please make the helpers in Uno.UI/Helpers `internal` only, so that from the public API perspective only the `Uno.UI.Toolkit` version is visible